gostatsd.go

```diff
@@ -67,8 +67,8 @@ func main() {
 			return
 		}
 		defer func() {
-			if err := f.Close(); err != nil {
-				augmentErr(&exitErr, fmt.Errorf("Failed to close profile file: %v", err))
+			if closeErr := f.Close(); closeErr != nil {
```

Comment:
We could just remove this profiling code. If/when we will want/need to do profiling we can always add it back (without committing) and use. Currently it looks like debugging statements that someone forgot to remove after a big debugging session :)